Sources/ARSAL_Mutex.c

```diff
@@ -145,11 +174,36 @@ int ARSAL_Mutex_Unlock(ARSAL_Mutex_t *mutex)
 #endif
             );
     }
+#elif defined(HAVE_WINDOWS_H)
+    result = ReleaseMutex((HANDLE)*mutex);
+    if (result != TRUE) {
+        int err = GetLastError();
+        ARSAL_PRINT(ARSAL_PRINT_FATAL, ARSAL_MUTEX_TAG,
+                    "Mutex unlock operation failed! GetLastError() = %d", err);
+        result = 1;
+    } else {
+        result = 0;
+    }
 #endif
 
     return result;
 }
 
+#ifdef HAVE_WINDOWS_H
```

Comment:
Condition variable are available sin Windows Vista and Windows Phone8:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms683469(v=vs.85).aspx
Available in mingw64 headers as well
Simply requires _WIN32_WINNT >= 0x0600 (#define _WIN32_WINNT 0x0600 before including windows header should work)
Unless there is a need to support old environment, it is simplier to use them